lib/python/qmk/keymap.py

```diff
@@ -371,7 +370,9 @@ def _c_preprocess(path, stdin=None):
     Returns:
         the stdout of the pre-processor
     """
-    pre_processed_keymap = qmk.commands.run(['cpp', path] if path else ['cpp'], stdin=stdin, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
+    cmd = ['cpp', path] if path else ['cpp']
```

Comment:
@fauxpark this is the line causing `qmk pytest` to fail on msys2. If I use `str(path)` instead it works fine.